tasks/1-text-2-speech/README.md

```diff
@@ -95,8 +95,8 @@ can also be found at **https://preview.golem.network/**
 
 ## Useful links:
 
-- Javascript docs: https://docs.golem.network/creators/javascript/quickstart/
-- Dapps docs: https://docs.golem.network/creators/dapps/ 
+- Javascript docs: https://docs.golem.network/docs/creators/javascript/quickstart/
```

Comment:
broken link, should be https://docs.golem.network/docs/creators/javascript/quickstarts/quickstart